app/controllers/development_metrics_controller.rb

```diff
@@ -48,7 +48,7 @@ def code_climate_project_summary
   def code_climate_department_summary
     CodeClimate::ProjectsSummaryService.call(
       department: department,
-      from: nil,
+      from: metric_params[:period],
```

Comment:
Do we really need to change the implementation of `CodeClimate::ProjectsSummaryService`? Wouldn't we just need to change parameter we're sending? If it already accepts a date, why don't we convert the input we have into a date?
We could just do something like
`from: metric_params[:period].weeks.ago,`
Or am I missing something?

Reply:
this implementation was done without counting the params that are handled in the frontend, the frontend always sent just an integer and since this sent just a `nil` it didn't crash, if you check the `CodeClimate::ProjectsDetailsService` you will notice the key `from` is an integer

Reply:
Alright, I'm okay with the intention of them matching.
I should note, though, that I think now the name `from` is not very expressive, since it seems to be expecting a date instead of a number of weeks for the look-back